### PR TITLE
[Neat-635] public examples

### PR DESCRIPTION
**Pull request by Juliamg**

### CODE REVIEW

File: cognite/neat/_session/_base.py [hunk 1]
```diff
@@ -92,6 +102,15 @@ def verify(self) -> IssueList:
...
         `.inspect.issues()` to see what went wrong.
+
+        Example:
+
+            # From an active NeatSession
+            ...
+            >>>neat.read.csv("your_csv_file.csv", type="example", primary_key="example_id")
+            >>>neat.infer()
+            >>>neat.verify()
+            'No issues found'
```
Comment (Juliamg) [inline cognite/neat/_session/_base.py:113]: I use this example format when there is an output message for a successful command. In this case "No issues found"

File: cognite/neat/_session/_drop.py [hunk 2]
```diff
@@ -23,6 +23,12 @@ def instances(self, type: str | list[str]) -> None:
 
         Args:
             type: The type of instances to drop.
+
+        Example:
```
Comment (Juliamg) [inline cognite/neat/_session/_drop.py:27]: This is the other format type which will appear in a code block in the docs:

**Comment by Juliamg** [comment]
Added an html file of the data model show output, but the formatting is weird and all links become hyperlinks on the page.. Not sure if it should be included in this PR until I have figured out a way to fix it?

File: cognite/neat/_session/_read.py [hunk 3]
```diff
@@ -283,6 +297,13 @@ class CSVReadAPI(BaseReadAPI):
         io: file path or url to the csv
         type: string that specifies what type of data the csv contains. For instance "Asset" or "Equipment"
         primary_key: string name of the column that should be used as the unique identifier for each row of data
+
+    Example:
+        ```python
+        type_described_in_table = "Turbine"
+        column_with_identifier = "UNIQUE_TAG_NAME"
+        neat.read.csv("url_or_path_to_csv_file", type=type_described_in_table, primary_key=column_with_identifier)
```
Comment (doctrino) [inline cognite/neat/_session/_read.py:305]: Nice use of variables to document.

**Review by doctrino — APPROVED** [review]
Good work!

**Pull request merged**